app/Models/WaitingRoom.php

```diff
@@ -29,6 +29,20 @@ public function users()
         return $this->belongsToMany(User::class);
     }
 
+    /**
+     * Determines if the waiting room is open for signupsOpen
+     *
+     * @return bool whether the room is open or not
+     */
+    public function isOpen()
+    {
+        $start = $this->signup_start_date->startOfDay();
+        $end = $this->signup_end_date->endOfDay();
+        $today = Carbon::now();
+
+        return ($today >= $start) && ($today <= $end);
```

Comment:
Did you try `return $today->between($start, $end)`? It should work for this and reads a lot nicer.